### DISCUSSION TITLE
Turn restriction is ignored

### DISCUSSION THREAD

**Discussion by Peter0** [osmandapp/OsmAnd#17043] [Maps Related]
When routing with internal car router, a route via node 2132522412 is ignoring the turn restriction https://www.openstreetmap.org/relation/2727821

E.g. from 50.73430 7.08827 to 50.73292 7.08749.
OSMand 4.3.12

**Comment by vshcherb** [comment]
Can't reproduce it with latest routing version - https://osmand.net/map?start=50.734017%2C7.087622&end=50.733354%2C7.087053&mode=car#19/50.73315/7.08804
